src/app/parcels/getExpandedParcelDetails.ts

```diff
@@ -82,9 +86,14 @@ export interface ExpandedParcelDetails extends Data {
     collection: string;
 }
 
-export const rawDataToExpandedParcelDetails = (
+export interface ExpandedParcelDetails {
+    expandedParcelData: ExpandedParcelData;
+    events: EventTableRow[];
```

Comment:
Was there any reason not to add `events` directly to the original `ExpandedParcelData`?

Reply:
Yes, expandedParcelData is passed into the DataViewer component, so I think the ExpandedParcelData type must extend the Data type, which only allows strings, arrays of strings, booleans, numbers and null. events needs to be an array of EventTableRow, so that we can easily pass it into the Table component. I decided it would be easiest to make a new type ExpandedParcelDetails to contain both expandedParcelData and events, so that we can still call the getExpandedParcelDetails function and get everything we need from that - mostly so that the database call for the event data can be made at the same time as the rest of the data. Is this ok?

Reply:
That's completely fair, thanks!